e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -75,6 +77,127 @@ describe('Integrations', () => {
+
+        // * Verify that that markdown isn't rendered
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#post_${postId}`).find('.user-popover').should('have.text', user1.username);
+            cy.get(`#postMessageText_${postId}`).get('code').should('have.text', '1. Not a list item, **not bolded**, http://notalink.com, ~off-topic is not a link to the channel.');
+        });
+    });
+
+    it('MM-T679 /echo', () => {
+        loginAndVisitChannel(user1, testChannelUrl);
+
+        // # Type "/echo test 3"
+        cy.postMessage('/echo test 3');
+
+        // * Verify that that post is not shown after 1 second
+        cy.wait(TIMEOUTS.ONE_SEC);
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('have.not.text', 'test');
+        });
+
+        // * Verify that that test 3 is posted after 3 seconds
+        cy.wait(TIMEOUTS.TWO_SEC);
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#post_${postId}`).find('.user-popover').should('have.text', user1.username);
+            cy.get(`#postMessageText_${postId}`).should('have.text', 'test');
+        });
+    });
+
+    it('MM-T680 /help', () => {
+        loginAndVisitChannel(user1, testChannelUrl, {
+            onBeforeLoad: (win) => {
+                cy.stub(win, 'open');
+            },
+        });
+
+        // # Type "/help"
+        cy.postMessage('/help');
+
+        // * Verify that a new tag opens
+        cy.window().its('open').should('be.called');
+    });
+
+    it('MM-T681 /invite_people error message with no text or text that is not an email address', () => {
+        loginAndVisitChannel(user1, testChannelUrl);
+
+        // # Type "/invite_people 123"
+        cy.postMessage('/invite_people 123');
+
+        // * Verify the message is shown saying "Please specify one or more valid email addresses"
+        verifyEphemeralMessage('Please specify one or more valid email addresses');
+    });
+
+    it('MM-T682 /leave', () => {
+        // # Go to Off-Topic
+        loginAndVisitChannel(user1, `${testTeam.name}/channels/off-topic`);
+
+        // # Type "/leave"
+        cy.postMessage('/leave');
+
+        // * Verity Off-Topic is not shown in LHS
+        cy.get('#sidebarChannelContainer').should('be.visible').should('not.contain', 'Off-Topic');
+
+        // * Verify user is redirected to Town Square
+        cy.get('#sidebarChannelContainer').should('be.visible').should('contain', 'Town Square');
+        cy.get('#channelHeaderTitle').should('be.visible').should('contain', 'Town Square');
+    });
+
+    it('MM-T683 /join', () => {
+        loginAndVisitChannel(user1, testChannelUrl);
+
+        // # Create New Channel
+        cy.apiCreateChannel(testTeam.id, 'new-channel', 'New Channel').then(({channel}) => {
+            // # Type "/join ~new-channel"
+            cy.postMessage(`/join ~${channel.name}`);
```

Comment:
This channel will be created by the `user1` itself and doing `/join` will not effectively do the command. I suggests below for other user to create a channel, and then be joined by `user1`
```suggestion
       // # Login as another user and create a new channel
        cy.apiAdminLogin();
        cy.apiCreateChannel(testTeam.id, 'new-channel', 'New Channel').then(({channel}) => {
            // # Login as test user and visit test channel
            loginAndVisitChannel(user1, testChannelUrl);

            // # Type "/join ~new-channel"
            cy.postMessage(`/join ~${channel.name}`);
```